assertk-common/src/test/kotlin/test/assertk/CatchSpec.kt

```diff
@@ -0,0 +1,29 @@
+package test.assertk
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+
+class CatchSpec_an_exception_On_catch {
+    private val subject = TestException()
+    @Test
+    fun it_should_catch_and_return_a_exception() {
```

Comment:
Not quite sure why we moved to snake case for test case names 😕

Reply:
Was a comment on the other pr, but apparently fancy characters like "spaces" don't work on multiplatform

Reply:
Indeed, spaces in function names is not supported on the javascript platform